hnn_core/drives.py

```diff
@@ -127,7 +129,8 @@ def _add_drives_from_params(net):
                 burst_std=specs['dynamics']['burst_std'],
                 numspikes=specs['dynamics']['numspikes'],
                 spike_isi=specs['dynamics']['spike_isi'],
-                repeats=specs['dynamics']['repeats'],
```

Comment:
wouldn't the historical param files contain `repeats` ? This won't break?

Reply:
The conversion from `repeats` to `n_drive_cells` keys happens in `params.py.create_pext()`. The param file itself contains `repeats`, but `specs` is a dict that gets formed by parsing the param file keys in `params.py.create_pext()` and `params.py._extract_drive_specs_from_hnn_params()`.